src/components/ActionLink/ActionLink.tsx

```diff
@@ -55,12 +55,14 @@ export type ActionLinkProps = NavigationActionLinkProps | ClickableActionLinkPro
 export default class ActionLink extends React.Component<ActionLinkProps> {
   public render() {
     const { ariaLabel, className, icon: Icon, text } = this.props;
-    const classNames = join(['y-actionLink', className]);
+    const classNames = getClassNames();
 
     const content = (
-      <FixedGridRow gutterSize={GutterSize.SMALL} className="y-actionLink--wrapper">
+      <FixedGridRow gutterSize={GutterSize.SMALL} className={classNames.wrapper}>
         <FixedGridColumn fixed={true}>
-          <Icon size={IconSize.MEDIUM} block={true} className="y-actionLink--icon" />
+          <Block push={2}>
+            <Icon size={IconSize.MEDIUM} block={true} />
+          </Block>
```

Comment:
I'm probably missing something here, I used a Block instead of 

```css
.y-actionLink--icon {	
  margin-top: 0.4rem;	

  /* Remove when this is addressed: https://github.com/Microsoft/YamUI/issues/327 */	
  position: static;	
  top: 0;	
}
```

Reply:
i think i left it as CSS because it was clear it needed to be updated when the issue was resolved, and lighter-weight solution. if you're going to move it to markup could you keep the comment with it?

Reply:
I tried css, but switched back again, because the style was losing out to the styles on `.y-icon`.  I presume this is because the merge-styles are being loaded before the css styles.  I've added the comment above the render function.